drivers/scheduler/k8s/specs/test-sharedv4/pod.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-sv4-dep-svc
```

Comment:
Everything in this app should be named test-sharedv4-dep, test-sharedv4-app etc to avoid confusion with the svc app.

Reply:
updated